scenarios.md

```diff
@@ -262,6 +261,20 @@ A user may want to trust an artifact only if it is signed by a specific key. May
 
 1. Users must be allowed to configure specific trusted keys for specific artifacts or collections of artifacts.
 
+### Scenario #11: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. The signature on the vulnerable artifact is revoked.
+1. Consumers that pulled the vulnerable artifact that verify the signature will receive an error indicating the signer no longer trusts the artifact.
+
+**Implications of this requirement:**
+
+1. Users that wish to still run artifacts that have their signatures revoked will need to disable verification on the vulnerable artifact or resign the artifacts with a key they control and trust.
```

Comment:
Consumers that set policy "on revocation - warning" will still be able to continue using the same artifact without a verification hard failure. Disabling verification would mean loosing other protections like integrity check and authenticity (publisher matches). If policy is "on revocation - error" , consumer could resign, or update to an unrevoked artifact.

Reply:
"disable verification" would mean different things depending on the client. At a high level to me it means they want to approve the artifact despite the failing Notary check. Whether that's a policy to disable a subset of the checks, a single artifact override, a "warn-only" setting, or something else I'm trying to avoid defining here.